proto/BUILD.bazel

```diff
@@ -29,13 +29,14 @@ go_proto_compiler(
 
 go_proto_compiler(
     name = "go_grpc",
-    options = ["plugins=grpc"],
+    plugin = "@org_golang_google_grpc_cmd_protoc_gen_go_grpc//:protoc-gen-go-grpc",
+    suffix = "_grpc.pb.go",
+    valid_archive = False,
```

Comment:
Defining a new, separate target would avoid issues with things like gazelle and allow incremental migration.